assets/scss/_alert.scss

```diff
@@ -0,0 +1,21 @@
+.alert {
```

Comment:
I wonder if it would be worth adjusting the link color within these alerts. I find it a little difficult to distinguish the link from regular text. With the current dev-site we at least increased font weight and had a dashed underline to help distinguish. 

<img width="850" alt="Screenshot 2024-05-30 at 12 06 21 PM" src="https://github.com/maxmind/dev-site/assets/52248962/8ae75343-329d-4f2b-92ad-7697a335ffdb">

Reply:
Hmm, yeah, I share the same concern.  I think the regular text color for the whole site is little too hard to read.  How do you feel about darkening all regular text instead of changing the link colour?

![image](https://github.com/maxmind/dev-site/assets/1154185/8f4195be-e8f8-4c7b-bb46-12f2b36a8234)

Reply:
Yes, this looks much better IMO!